systemd/systemd.go

```diff
@@ -1468,13 +1466,13 @@ func (s *systemd) EnsureMountUnitFile(snapName, revision, what, where, fstype st
 		hostFsType = "none"
 	}
 	return s.EnsureMountUnitFileWithOptions(&MountUnitOptions{
-		Lifetime: Persistent,
-		SnapName: snapName,
-		Revision: revision,
-		What:     what,
-		Where:    where,
-		Fstype:   hostFsType,
-		Options:  options,
+		Lifetime:    Persistent,
+		SnapName:    snapName,
+		Description: revision,
```

Comment:
shouldn't the argument name also change?

Reply:
Done